build.sbt

```diff
@@ -64,13 +56,16 @@ lazy val microservice = Project(appName, file("."))
       "-Yrangepos",
       "-language:postfixOps"
     ),
-    scalacOptions in Test --= Seq("-Ywarn-value-discard")
+    Test / scalacOptions --= Seq("-Ywarn-value-discard")
   )
   .settings(scalacOptions ++= Seq("-Yrangepos", "-Ywarn-unused:imports"))
   .settings(publishingSettings: _*)
   .configs(IntegrationTest)
   .settings(integrationTestSettings(): _*)
-  .settings(resolvers ++= Seq(Resolver.jcenterRepo, "emueller-bintray" at "http://dl.bintray.com/emueller/maven"))
+  .settings(resolvers ++= Seq(
+    Resolver.jcenterRepo,
+    ("emueller-bintray" at "http://dl.bintray.com/emueller/maven").withAllowInsecureProtocol(true)
```

Comment:
can we use https and remove the allow thing? Or they don't have it?

Reply:
Am not able to use https here. Getting lots of compilation issues.